exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/DrillFuncHolder.java

```diff
@@ -262,6 +260,78 @@ protected void addProtectedBlock(ClassGenerator<?> g, JBlock sub, String body, H
     }
   }
 
+  /**
+   * Checks whether current function is var arg one and declares array for storing vararg function arguments.
+   *
+   * @param model          code model to generate the code
+   * @param jBlock         block of code to be populated
+   * @param inputVariables array of input variables for current function
+   */
+  protected void declareVarArgArray(JCodeModel model, JBlock jBlock, HoldingContainer[] inputVariables) {
+    if (isVarArg()) {
```

Comment:
Not sure if this check should be done here, I would assume its caller responsibility first check `isVarArg()` and then call `declareVarArgArray`.